bin/inx-chronicle/src/api/v1/extractors.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use async_trait::async_trait;
+use axum::extract::{FromRequest, Query};
+use hex::FromHex;
+use serde::{Deserialize, Serialize};
+
+use crate::api::error::APIError;
+
+#[derive(Clone, Default, Debug, Serialize, Deserialize)]
```

Comment:
Just making sure: the default for `included` should be `false` for `MessagesQuery` but `true` for `OutputsQuery`?

Reply:
Hmm. No, they should probably both be true.